lib/packages/core/pubspec.yaml

```diff
@@ -40,6 +40,8 @@ dependencies:
   custom_pop_up_menu: ^1.2.4
   bloc: ^8.1.4
   meta: ^1.11.0
+  http: ^1.2.2
```

Comment:
we can use dio package itself, instead of http as we are already using dio in data layer. it will be better.

Reply:
We just need to make a simple http request, didn't want any bells and whistles. Plus the code when it was uplifted depended on `http`. So it would have been extra work to remove it.

Reply:
Here the use case is simple but we should not depend upon multiple package for the same functionality, that's why i asked you to remove `image_picker` dependency recently. The dependency migration from http to dio won't take much effort since it is only used in a single file only. We will do the same change in Feed-SDK  also whenever we will push a new release.

Reply:
Then make a note of that, and remember to do it. And I image_picker I asked us to remove because it wasn't being used and file_pciker did a better job. For a simple use of making a http request - I don't see the point in using dio. We use dio because we wanted a lot of customisation in the way requests and responses were being made.